File: Data Structures & Algorithms/majority-element/submission-1.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        unordered_map<int, int> frequency;
        int size = nums.size();
        for (int num: nums) {
            frequency[num] = frequency[num] + 1;
            if(frequency[num] > size/2) {
                return num;
            }
        }
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        numHash = {}
        if nums == []:
            return 0
        for num in nums:
            numHash[num] = set()
        
        for num in nums:
            curr_num = num
            while curr_num + 1 in numHash:
                numHash[num].add(curr_num + 1)
                curr_num = curr_num + 1
        lengths = [len(x) + 1 for x in list(numHash.values())]
        return max(lengths)
                
